awx/main/tasks/system.py

```diff
@@ -615,8 +615,10 @@ def awx_receptor_workunit_reaper():
     jobs_with_unreleased_receptor_units = UnifiedJob.objects.filter(work_unit_id__in=unit_ids).exclude(status__in=ACTIVE_STATES)
     for job in jobs_with_unreleased_receptor_units:
         logger.debug(f"{job.log_format} is not active, reaping receptor work unit {job.work_unit_id}")
-        receptor_ctl.simple_command(f"work cancel {job.work_unit_id}")
-        receptor_ctl.simple_command(f"work release {job.work_unit_id}")
+        try:
+            receptor_ctl.simple_command(f"work release {job.work_unit_id}")
+        except Exception as exc:
+            logger.error(f'Failed to reap {job.log_format} unit_id={job.work_unit_id}, error: {exc}')
```

Comment:
@fosterseth I think this came up in some case you were debugging lately. It's been known for a long while that we need to fix this error handling. We could break this out from the rest of the changes here to get it in faster.